crates/ra_syntax/tests/data/parser/ok/0044_let_attrs.txt

```diff
@@ -0,0 +1,73 @@
+SOURCE_FILE@[0; 166)
+  FN_DEF@[0; 165)
+    COMMENT@[0; 60)
+    WHITESPACE@[60; 61)
+    FN_KW@[61; 63)
+    WHITESPACE@[63; 64)
+    NAME@[64; 68)
+      IDENT@[64; 68) "main"
+    PARAM_LIST@[68; 70)
+      L_PAREN@[68; 69)
+      R_PAREN@[69; 70)
+    WHITESPACE@[70; 71)
+    BLOCK@[71; 165)
+      L_CURLY@[71; 72)
+      WHITESPACE@[72; 77)
+      ATTR@[77; 106)
+        POUND@[77; 78)
+        TOKEN_TREE@[78; 106)
+          L_BRACK@[78; 79)
+          IDENT@[79; 82) "cfg"
+          TOKEN_TREE@[82; 105)
+            L_PAREN@[82; 83)
+            IDENT@[83; 90) "feature"
+            WHITESPACE@[90; 91)
+            EQ@[91; 92)
+            WHITESPACE@[92; 93)
+            STRING@[93; 104)
+            R_PAREN@[104; 105)
+          R_BRACK@[105; 106)
+      WHITESPACE@[106; 111)
+      LET_STMT@[111; 163)
```

Comment:
I think in this case an `ATTR` should be a child of `LET_STMT`. 

Do do this, I think we need to pull `attributes::outer_attributes(p);` *out* of `maybe_item`, and force the caller to call `outer_attributes`. That way, we can reuse attrs for either item or let.